src/renderer.zig

```diff
@@ -851,6 +857,15 @@ pub const MeshSelection = struct { // MARK: MeshSelection
 
 	pub fn breakBlock(inventory: main.items.Inventory, slot: u32, deltaTime: f64) void {
 		if(selectedBlockPos) |selectedPos| {
+			const stack = inventory.getStack(slot);
+			const isSelectionWand = stack.item != null and stack.item.? == .baseItem and std.mem.eql(u8, stack.item.?.baseItem.id, "cubyz:selection_wand");
+			if(isSelectionWand) {
+				std.log.debug("Position 1: {}", .{selectedPos});
```

Comment:
Do we need the debug prints? After all it's displayed visually anyways, and no one is going to read the logs for this,

Reply:
No, probably we can get rid of them, since now it works.